uniswap_swap_bot/src/bot.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+const handleTransaction: HandleTransaction = async (
```

Comment:
The team has a standard of naming these functions `provideHandle...`, and this would be `provideHandleTransaction`. Take a look at our bot implementations in our repo to know what I'm referring to.